test/manifests.go

```diff
@@ -56,12 +56,20 @@ const ManifestJSON string = `{
 			"Parameters": {
 				"Files": {
 					"/tmp/defg.txt": "foo",
-					"/tmp/jkl.mno": "bar"
+					"/tmp/jkl.mno": "bar",
+					"/tmp/base64.txt": {
+						"Data": "TWFyYmxlUnVuIGJhc2U2NA==",
+						"Encoding": "base64",
+						"NoTemplates": true
+					}
 				},
 				"Env": {
 					"IS_FIRST": "true",
 					"SEAL_KEY": "{{ hex .Marblerun.SealKey }}",
-					"TEST_SECRET_SYMMETRIC_KEY": "{{ raw .Secrets.symmetric_key_shared }}",
+					"TEST_SECRET_SYMMETRIC_KEY": {
+						"Data": "{{ raw .Secrets.symmetric_key_shared }}",
```

Comment:
Given that the code now explicitly checks against NULL bytes passed to the environment, is this supposed to fail? Actually not right now regarding the tests if we check this or we don't.

Reply:
This should be replaced with a different key, probably `base64` or `hex`